src/net/addr.rs

```diff
@@ -0,0 +1,302 @@
+use crate::future::Future;
+use crate::task::blocking::JoinHandle;
+use futures::future::{ready, Ready};
+use std::net::{IpAddr, Ipv4Addr, Ipv6Addr};
+pub use std::net::{SocketAddr, SocketAddrV4, SocketAddrV6};
+
+/// A trait for objects which can be converted or resolved to one or more
+/// [`SocketAddr`] values.
```

Comment:
This link is missing an URL.

Reply:
I've reexport `SocketAddr`, so maybe the URL is not required? (I've tried `cargo doc` and the link isn't broken.)